e2e/cypress/integration/messaging/focus_move_spec.js

```diff
@@ -149,12 +149,6 @@ describe('Messaging', () => {
         });
     });
     it('M17455 - Focus does not move for non-character keys', () => {
-        //# Make sure main input is visible and focused
-        cy.get('#post_textbox').should('be.visible').and('be.focused');
```

Comment:
Why remove?  I thought the focus should be on the center textbox on visit to a channel.

Reply:
For some reason, loading the page manually on browser puts focus on center textbox correctly. However, in cypress, the focus never happens. I don't think it's an issue with the app since it works manually. The test case seems to just verify focus hasn't changed (or in this case, lack of focus). I can put it back and let it fail if you think otherwise.

Reply:
That makes sense.  I've checked the [test case](https://docs.google.com/spreadsheets/d/1-3usQeVbXS798rNt2IkcsKM-KxcckLnsehztSkeSiNw/edit#gid=1636776662) and we're still align on it.
> 1. Left click on a post to move the focus out of the main input box
> 2. Hit the following keys and watch to see if the focus is returned to the main input box: Num Lock, down arrow, Shift, F7, Pg Up(from numpad with num lock off), enter and a system key such as win or fn